guide/source/prepare-meteor-3.0.md

```diff
@@ -0,0 +1,121 @@
+---
+title: How to prepare for Meteor Async (3.0)
```

Comment:
I think this title gives the impression that this guide is for Meteor 3.0.

We could change it to something like:
```suggestion
title: How to migrate to Meteor Async in Meteor 2.x
```